base/applications/mspaint/canvas.cpp

```diff
@@ -268,9 +268,18 @@ LRESULT CCanvasWindow::OnLRButtonDown(BOOL bLeftButton, UINT nMsg, WPARAM wParam
     HITTEST hitSelection = SelectionHitTest(pt);
     if (hitSelection != HIT_NONE)
     {
+        selectionModel.m_nSelectionBrush = 0; // Selection Brush is OFF
         if (bLeftButton)
         {
             CanvasToImage(pt);
+            if (::GetKeyState(VK_CONTROL) < 0) // Selection Stamp
```

Comment:
What is "Selection stamp" about?

Reply:
The image of stamp:
![image](https://github.com/reactos/reactos/assets/2107452/0f28c994-8fab-4013-8265-aebb6145b925)

Reply:
I had to test the thing in mspaint in order to understand what you mean. Maybe better to call that "Selection Clone" or sthg like that, since it copies the selection to where it is currently present. I don't know whether there is an "official" naming of this feature.

Reply:
Renamed.